src/compile/layout.ts

```diff
@@ -72,7 +72,9 @@ export function parseUnitLayout(model: UnitModel): LayoutComponent {
 }
 
 function parseUnitSizeLayout(model: UnitModel, channel: Channel): SizeComponent {
-  const nonOrdinalSize = channel === X ? model.cellWidth() : model.cellHeight();
+  // TODO: think about whether this config has to be the cell or facet cell config
```

Comment:
always cell config, and unit inside facet should have its config overridden by facet?